pkg/types/cose/v0.0.1/cose_v0_0_1_schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/intoto/intoto_v0_0_1_schema.json",
+    "title": "cose v0.0.1 Schema",
+    "description": "Schema for cose object",
+    "type": "object",
+    "properties": {
+        "message": {
+            "type": "string",
+            "format": "byte",
+            "description": "The COSE Sign1 Message"
+        },
+        "publicKey": {
+            "description": "The public key that can verify the signature",
```

Comment:
Does COSE support certificates in addition to public keys? `x509.NewPublicKey` supports either.

Reply:
https://datatracker.ietf.org/doc/html/rfc7517#page-9

Reply:
In the rekor data model it would support certs, where we explicitly require the certificate to be listed out of band. I don't think x5c is supported inline though as @OR13 points out.

Reply:
**As per section 11.11:**
The formatting and processing for c5b, c5c, and c5t, and c5u are similar to x5bag, x5chain, x5t, x5u defined in  [I-D.ietf-cose-x509]